openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -1134,6 +1138,160 @@ def parse_smirnoff_from_source(self, source):
         msg += f"Parsing failed with the following error:\n{exception_msg}\n"
         raise IOError(msg)
 
+    def _terms(self):
+        """
+        Generate a list of active term keys. These keys are can be used to find the
+        specific value. The order the keys will be the same, and will only differ
+        if the order is manually changed, or the list of active terms is changed.
```

Comment:
The terminology I am adhering to says that a parameter is a list of terms. A term can also be values not ascribed to a parameter, e.g. scale14. So, I think these keys will never point to a parameter, but a term of a parameter (or handler).

Reply:
Okay, that makes sense. I think that meaning should be written up in this docstring, currently it would be confusing to a new reader what the values are. Maybe something like

>  These keys are can be used to find the specific value of an associated term.

and also define "term" elsewhere (top of `_ParameterTermKey`?)